From 6b09c0348113c5412c12bf92542e5e7b417bbb8c Mon Sep 17 00:00:00 2001
From: Valentine Barshak <valentine.barshak@cogentembedded.com>
Date: Mon, 5 Oct 2020 22:33:59 +0300
Subject: [PATCH 17/23] arm64: dts: renesas: r8a779[78]0: Update RPC device
 nodes

This updates RPC device nodes for the consolidated RPC driver.

Signed-off-by: Valentine Barshak <valentine.barshak@cogentembedded.com>
---
 arch/arm64/boot/dts/renesas/r8a77970-eagle.dts  | 2 +-
 arch/arm64/boot/dts/renesas/r8a77970-v3msk.dts  | 2 +-
 arch/arm64/boot/dts/renesas/r8a77970.dtsi       | 4 ++--
 arch/arm64/boot/dts/renesas/r8a77980-condor.dts | 2 +-
 arch/arm64/boot/dts/renesas/r8a77980-v3hsk.dts  | 2 +-
 arch/arm64/boot/dts/renesas/r8a77980.dtsi       | 4 ++--
 6 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a77970-eagle.dts b/arch/arm64/boot/dts/renesas/r8a77970-eagle.dts
index 34b2d83..969308b 100644
--- a/arch/arm64/boot/dts/renesas/r8a77970-eagle.dts
+++ b/arch/arm64/boot/dts/renesas/r8a77970-eagle.dts
@@ -398,7 +398,7 @@
 	};
 };
 
-&qspi0 {
+&rpc0 {
 	pinctrl-0 = <&qspi0_pins &qspi1_pins>;
 	pinctrl-names = "default";
 
diff --git a/arch/arm64/boot/dts/renesas/r8a77970-v3msk.dts b/arch/arm64/boot/dts/renesas/r8a77970-v3msk.dts
index c20f0c2..a897c4a 100644
--- a/arch/arm64/boot/dts/renesas/r8a77970-v3msk.dts
+++ b/arch/arm64/boot/dts/renesas/r8a77970-v3msk.dts
@@ -259,7 +259,7 @@
 	};
 };
 
-&qspi0 {
+&rpc0 {
 	pinctrl-0 = <&qspi0_pins &qspi1_pins>;
 	pinctrl-names = "default";
 
diff --git a/arch/arm64/boot/dts/renesas/r8a77970.dtsi b/arch/arm64/boot/dts/renesas/r8a77970.dtsi
index 5edf853..8c7e101 100644
--- a/arch/arm64/boot/dts/renesas/r8a77970.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a77970.dtsi
@@ -766,8 +766,8 @@
 			status = "disabled";
 		};
 
-		qspi0: spi@ee200000 {
-			compatible = "renesas,qspi-rpc-r8a77970";
+		rpc0: spi@ee200000 {
+			compatible = "renesas,rpc-r8a77970";
 			reg = <0 0xee200000 0 0x1f0>,
 			      <0 0x08000000 0 0x04000000>,
 			      <0 0xee208000 0 0x100>;
diff --git a/arch/arm64/boot/dts/renesas/r8a77980-condor.dts b/arch/arm64/boot/dts/renesas/r8a77980-condor.dts
index 20fd84e..8bcc78a 100644
--- a/arch/arm64/boot/dts/renesas/r8a77980-condor.dts
+++ b/arch/arm64/boot/dts/renesas/r8a77980-condor.dts
@@ -609,7 +609,7 @@
 	};
 };
 
-&qspi0 {
+&rpc0 {
 	pinctrl-0 = <&qspi0_pins &qspi1_pins>;
 	pinctrl-names = "default";
 
diff --git a/arch/arm64/boot/dts/renesas/r8a77980-v3hsk.dts b/arch/arm64/boot/dts/renesas/r8a77980-v3hsk.dts
index 8a6b2ea..59182e8 100644
--- a/arch/arm64/boot/dts/renesas/r8a77980-v3hsk.dts
+++ b/arch/arm64/boot/dts/renesas/r8a77980-v3hsk.dts
@@ -275,7 +275,7 @@
 	};
 };
 
-&qspi0 {
+&rpc0 {
 	pinctrl-0 = <&qspi0_pins &qspi1_pins>;
 	pinctrl-names = "default";
 
diff --git a/arch/arm64/boot/dts/renesas/r8a77980.dtsi b/arch/arm64/boot/dts/renesas/r8a77980.dtsi
index a9f89ae..f701fdf 100644
--- a/arch/arm64/boot/dts/renesas/r8a77980.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a77980.dtsi
@@ -821,8 +821,8 @@
 			status = "disabled";
 		};
 
-		qspi0: spi@ee200000 {
-			compatible = "renesas,qspi-rpc-r8a77980";
+		rpc0: spi@ee200000 {
+			compatible = "renesas,rpc-r8a77980", "renesas,rpc";
 			reg = <0 0xee200000 0 0x1f0>,
 			      <0 0x08000000 0 0x04000000>,
 			      <0 0xee208000 0 0x100>;
-- 
2.7.4

